requirements/base.txt

```diff
@@ -63,4 +63,4 @@ sphinxcontrib-websupport==1.1.0  # via sphinx
 unidecode==1.0.22         # via vng-api-common
 uritemplate==3.0.0        # via coreapi, drf-yasg
 urllib3==1.24.1           # via requests
-vng-api-common[notifications]==0.38.0
+vng-api-common==0.46.2
```

Comment:
dependencies always need to be updated in the `requirements/*.in` files, after which you compile them down to the pinned versions with [`pip-compile` from pip-tools](https://pypi.org/project/pip-tools/).

There's a wrapper script to take care of all this: `./bin/compile_dependencies.sh`